docs/topics/cross-platform-mobile-development.md

```diff
@@ -0,0 +1,163 @@
+
+These issues raise the question of whether the end-user will notice a difference between native and cross-platform apps.
+
+As modern multiplatform frameworks continue to evolve, they increasingly allow mobile engineers to craft a native-like experience. If an application is well written, the user will not be able to notice the difference. However, the quality of your product will heavily depend on the cross-platform app development tools you choose.
+
+## The most popular cross-platform solutions
+
+The most popular cross-platform frameworks include Flutter, React Native, and Kotlin Multiplatform Mobile. Each of these frameworks has its own capabilities and strengths. Depending on the tool you use, your development process and the outcome may vary.
+
+### Flutter
+
+Flutter is a cross-platform development framework that was created by Google and uses the Dart programming language. Flutter supports native features, such as location services, camera functionality, and hard drive access. If you need to create a specific app feature that’s not supported in Flutter, you can write platform-specific code using the [Platform Channel technology](https://brightmarbles.io/blog/platform-channel-in-flutter-benefits-and-limitations/).
+
+Apps built with Flutter need to share all of their UX and UI layers, which is why they may not always feel 100% native. One of the best things about this framework is its Hot Reload feature, which allows developers to make changes and view them instantly.
+
+This framework may be the best option in the following situations:
+
+* You want to share UI components between your apps but you want your applications to look close to native.
+* The app is expected to put a heavy load on CPU/GPU.
+* You need to develop an MVP application.
+
+Among the most popular apps built with Flutter are Google Ads, Xianyu by Alibaba, eBay Motors, and Hamilton.
+
+### React Native
+
+Facebook introduced React Native in 2015 as an open-source framework designed to help mobile engineers build hybrid native/cross-platform apps. It’s based on ReactJS – a JavaScript library for building user interfaces. In other words, it uses JavaScript to build mobile apps for Android and iOS systems.
+
+React Native provides access to several third-party UI libraries with ready-to-use components, helping mobile engineers save time during the development process. Like Flutter, it allows you to see all your changes immediately, thanks to the Fast Refresh feature.
+
+You should consider using React Native for your app in the following cases:
+
+* Your application is relatively simple and is expected to be lightweight.
+* The development team is fluent in JavaScript or React.
+
+Applications built with React Native include Facebook, Instagram, Skype, Airbnb, and Uber Eats.
```

Comment:
AFAIK, Airbnb reverted back from React Native to native apps

Reply:
I think you're right. I've Googled it and found a few recent resources that said that Airbnb stopped using React Native.